pyccl/halos/halo_model.py

```diff
@@ -579,13 +608,28 @@ def pkf(sf):
             pk_2h = 0.
 
         if get_1h:
-            pk_1h = hmc.I_0_2(cosmo, k_use, aa, prof,
-                              prof_2pt, prof2)
+            pk_1h = hmc.I_0_2(cosmo, k_use, aa, prof, prof_2pt, prof2)
+            if supress_1h is not None:
+                ks = supress_1h(aa)
+                pk_1h *= (k_use / ks)**4 / (1 + (k_use / ks)**4)
         else:
             pk_1h = 0.
 
-        # Total power spectrum
-        out[ia, :] = (pk_1h + pk_2h) * norm
+        # Transition region
+        if smooth_transition is None:
+            out[ia, :] = (pk_1h + pk_2h) * norm
+        else:
+            alpha = smooth_transition(aa)
+            if alpha < 0.4 and not flag_alpha:
```

Comment:
Why 0.4 in particular?

Reply:
I think I explain the rationale of the warning, along with the value I've chosen, in my reply below.